packages/runner/src/lib/FunctionScanTarget.ts

```diff
@@ -0,0 +1,25 @@
+import fastify from 'fastify';
+
+export class FunctionScanTarget {
+  private readonly server = fastify();
+
+  public async start<T>(
+    fn: (input: T) => Promise<unknown>
+  ): Promise<{ url: string }> {
+    this.server.post('/', async (request, reply) => {
```

Comment:
I suggest to set error handler using `setErrorHandler` to avoid misbehavior in case of HTTP method fuzzing, insecure HTTP message parsing, etc.